### PR TITLE
Add Sort Functionality

### LINKED ISSUE #118: [Feature] Sort
Implementing  feature sorting by name and status
Sort by name ascending, prefix: sna/
Sort by name descending, prefix: snd/
Sort by status ascending, prefix: ssa/
Sort by status descending, prefix: ssd/
Sort by name then status, prefix: so/

# Tasks
- [x] Implement sort feature
- [x] Update UG and DG

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/seedu/trackermon/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -9,5 +9,10 @@
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_STATUS = new Prefix("s/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_SORT_NAME_ACS = new Prefix("sna/");
+    public static final Prefix PREFIX_SORT_STATUS_ACS = new Prefix("ssa/");
+    public static final Prefix PREFIX_SORT_NAME_DES = new Prefix("snd/");
+    public static final Prefix PREFIX_SORT_STATUS_DES = new Prefix("ssd/");
+    public static final Prefix PREFIX_SORT_STATUS_ORD = new Prefix("so/");
```
Comment (JonathanHoshi) [inline src/main/java/seedu/trackermon/logic/parser/CliSyntax.java:16]: Just curious, what is `PREFIX_SORT_STATUS_ORD`?

**Review by JonathanHoshi — CHANGES_REQUESTED** [review]
Good job! Other than the few naming issues, LGTM!

**Review by Ardentsoul — APPROVED** [review]
LGTM!

File: src/main/java/seedu/trackermon/model/Model.java [hunk 2]
```diff
@@ -45,43 +49,58 @@
     void setShowListFilePath(Path addressBookFilePath);
 
     /**
-     * Replaces address book data with the data in {@code addressBook}.
+     * Replaces show list data with the data in {@code showList}.
      */
     void setShowList(ReadOnlyShowList addressBook);
```
[the remark this answers is not in the document]
Comment (lowkaiwei98) [reply src/main/java/seedu/trackermon/model/Model.java]: opps, missed this one.

File: src/main/java/seedu/trackermon/model/Model.java [hunk 3]
```diff
@@ -45,43 +49,58 @@
     void setShowListFilePath(Path addressBookFilePath);
```
[the remark this answers is not in the document]
Comment (lowkaiwei98) [reply src/main/java/seedu/trackermon/model/Model.java]: opps, missed this one.

File: src/main/java/seedu/trackermon/logic/parser/CliSyntax.java (hunk 1 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/trackermon/logic/parser/CliSyntax.java:16]: For sorting status then name if both name and status are being sorted at the same time

Comment (JonathanHoshi) [reply src/main/java/seedu/trackermon/logic/parser/CliSyntax.java:16]: Shouldn't sorting name and status be just putting 2 of the prefix together?

**Review by JonathanHoshi — APPROVED** [review]
LGTM! Thanks for clarifying the doubts!

**Pull request merged**